website/docs/docs/running-a-dbt-project/using-the-dbt-ide.md

```diff
@@ -16,23 +16,23 @@ New dbt Cloud accounts will automatically be created with a Development Environm
 
 To create a Development Environment, navigate to the Environments page for your Project and click the "New Environment" button.
 
-<Lightbox src="/img/docs/running-a-dbt-project/0d9f366-Screen_Shot_2019-11-19_at_12.13.28_PM.png" title="Creating a new environment for the Analytics project"/>
+<Lightbox src="/img/docs/running-a-dbt-project/using-the-dbt-ide/empty-env-page.png" title="Creating a new environment for the Analytics project"/>
```

Comment:
there's a mismatch here between the screenshot and the paragraph. looks like this screenshot is of the Environments page under the Deploy menu (which has the **Create** button). and the paragraph is referring to this Environments page from the IDE (which has the **New** button and i believe is the one we want): 

![Screen Shot 2022-07-27 at 12 38 49 PM](https://user-images.githubusercontent.com/107218380/181357978-9eba3ff5-e9ee-4f2f-8050-5ec519bc5fc5.png)

Reply:
You're right. I'm referring to the  I think the text is wrong. It should say "Create Environment"


![Screen Shot 2022-07-27 at 2 31 06 PM](https://user-images.githubusercontent.com/39397591/181376067-89fa1ecf-e9e9-49ac-90fe-f894ea109c59.png)

Reply:
i'm still unsure. the steps in this section after clicking the button talk about changing the environment type field to **Development** but that's not possible from "Create environment" (it's stuck on **Deployment** for me): 

![Screen Shot 2022-07-27 at 2 43 57 PM](https://user-images.githubusercontent.com/107218380/181379084-3146d44d-850b-486c-b1ff-ec1cdf814fcd.png)

![Screen Shot 2022-07-27 at 2 53 02 PM](https://user-images.githubusercontent.com/107218380/181379265-95a4a19d-69a8-4c54-9d4e-dd7ddc9ab88e.png)

is my dbt Cloud account not using the latest from the UI refresh?

Reply:
You're in the right page --- you can only have one Development environment. But you can have multiple Deployment environment. So if you already an existing Development environement, then you can't add more.

Reply:
oh i see! thanks for the clarification, @bossnunta !